### ISSUE TITLE
doubt about boundary.png

### ISSUE THREAD

**Issue by zuixiaosanlang** [Dong-Huo/RGB-T-Glass-Segmentation#2]
boundary.png seems useless in the project ??

![image](https://user-images.githubusercontent.com/20105627/182336646-0a62fff6-bfb4-4564-b8d6-778354de4286.png)

loss only use bce_loss

**Comment by Dong-Huo** [comment]
That's right. As mentioned in the paper, we only used bce_loss. boundary.png is just generated in case.

**Comment by zuixiaosanlang** [comment]
@Dong-Huo 
Hi, 
So, can i give the same boundary.png when i only use rgb to infer ??

**Comment by Dong-Huo** [comment]
> @Dong-Huo Hi, So, can i give the same boundary.png when i only use rgb to infer ??

The boundary.png is never used for both rgb-t and rgb-only, you can use anything to replace it.

**Comment by zuixiaosanlang** [comment]
@Dong-Huo 
Hi, 
i want to change the model to onnx.
but find NestedTensor which not supported.
any suggest ？？

**Comment by Dong-Huo** [comment]
> @Dong-Huo Hi, i want to change the model to onnx. but find NestedTensor which not supported. any suggest ？？

I have little experience of onnx. Sorry about that...

**Comment by zuixiaosanlang** [comment]
@Dong-Huo 
Hello,
only use rgb to infer, but found the model alse need mask.
can i give the same mask ?
 example mask=torch.ones(.....
